CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 #### Additions
 * The speed of the Ancient Altar can now be configured in the `Items.yml` file
 * The message "You do not have enough knowledge to understand this" now includes the name of the item you need to research.
+* (API) Added EntityHitHandler
```

Comment:
Revert changes to changelog.md, these should be left for whoever compiles the new RC (cough cough Cookie)